DataConnectors/NXLogDnsLogs.json

```diff
@@ -0,0 +1,90 @@
+{
+    "id": "NXLogDnsLogs",
+    "title": "NXLog DNS Logs",
+    "publisher": "NXLog",
+    "descriptionMarkdown": "The NXLog DNS Logs data connector uses Event Tracing for Windows ([ETW](https://docs.microsoft.com/en-us/windows/apps/trace-processing/overview)) for collecting both Audit and Analytical DNS Server events. The [NXLog *im_etw* module](https://nxlog.co/documentation/nxlog-user-guide/im_etw.html) reads event tracing data directly for maximum efficiency, without the need to capture the event trace into an .etl file. This REST API connector can forward DNS Server events to Azure Sentinel in real-time.",

```

Comment:
The document link needs to have locale removed from the URL - \en-us\ - should be removed, otherwise validation fails.

Reply:
@shainw The data connector [NXLogDnsLogs.json](https://github.com/techwriter-dev/Azure-Sentinel/blob/master/DataConnectors/NXLogDnsLogs.json) was updated in [commit ded24573](https://github.com/techwriter-dev/Azure-Sentinel/commit/ded245731b8f9dbd3ac4c1f00544abfd77ab3469). Thanks!